.github/workflows/test.yml

```diff
@@ -9,9 +9,11 @@ on:
   pull_request:
 
 env:
+  PRIVATE_KEY_ANVIL: ${{ secrets.PRIVATE_KEY_ANVIL }}
   PRIVATE_KEY_SEPOLIA: ${{ secrets.PRIVATE_KEY_SEPOLIA }}
   PROVIDER_URI_SEPOLIA: ${{ secrets.PROVIDER_URI_SEPOLIA }}
   PROVIDER_URI_GOERLI: ${{ secrets.PROVIDER_URI_GOERLI }}
+  PROVIDER_URI_84532: ${{ secrets.PROVIDER_URI_84532 }}
```

Comment:
Why use the chainID instead of name here?

Reply:
this is part of the beginning of a broader change across the SDK repos where we can more easily keep provider URIs in line w/ chain IDs and reduce confusion/errors as we onboard more chains.

Reply:
note that this only affects our tests/CI

Reply:
Sounds good